src/pudl/metadata/codes.py

```diff
@@ -2318,6 +2318,212 @@
+                    "Hybrid (non-specified)",
+                ),
+                (
+                    "HRC",
+                    "hybrid_wet_dry",
+                    "Hybrid: cooling pond(s) or canal(s) with dry cooling",
+                ),
+                (
+                    "HRF",
+                    "hybrid_forced_draft_dry",
+                    "Hybrid: forced draft cooling tower(s) with dry cooling",
+                ),
+                (
+                    "HRI",
+                    "hybrid_induced_draft_dry",
+                    "Hybrid: induced draft cooling tower(s) with dry cooling",
+                ),
+                (
+                    "O",
+                    "once_through_non_specified",
+                    "Once through (non-specified)",
+                ),
+                (
+                    "O + R",
+                    "once_through_non_specified_and_recirculating_non_specified",
+                    "Once through (non-specified) and Recirculating (non-specified)",
+                ),
+                (
+                    "OC",
+                    "once_with_pond",
+                    "Once through with cooling pond(s)",
+                ),
+                (
+                    "OF",
+                    "once_fresh",
+                    "Once through, fresh water",
+                ),
+                (
+                    "ON",
+                    "once_without_pond",
+                    "Once through without cooling pond(s)",
+                ),
+                (
+                    "OS",
+                    "once_saline",
+                    "Once through, saline water",
+                ),
+                (
+                    "R",
+                    "recirculating_non_specified",
+                    "Recirculating (non-specified)",
+                ),
+                (
+                    "RC",
+                    "recirculating_pond",
+                    "Recirculating with cooling pond(s) or canal(s)",
+                ),
+                (
+                    "RF",
+                    "recirculating_forced_draft",
+                    "Recirculating with cooling pond(s) or canal(s)",
+                ),
+                (
+                    "RI",
+                    "recirculating_induced_draft",
+                    "Recirculating with induced draft cooling tower(s)",
+                ),
+                (
+                    "RN",
+                    "recirculating_natural_draft",
+                    "Recirculating with natural draft cooling tower(s)",
+                ),
+                (
+                    "HT",
+                    "helper_tower",
+                    "Helper tower",
+                ),
+                (
+                    "OT",
+                    "other",
+                    "Other",
+                ),
+            ],
+        ),
+        "code_fixes": {
+            "RECIRCULATING WITH INDUCED DRAFT COOLING TOWER": "RI",
```

Comment:
EIA 923 has the long versions of these codes in the data files themselves.

They're not *exactly* the same as the descriptions above, and also have some whitespace/casing issues, so I canonicalize them to uppercase in the transform and use this `code_fixes` dictionary to map them correctly.